content/docs/connectors/table/kudu/overview.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Overview
+weight: 1
+type: docs
+aliases:
+- /docs/connectors/table/kudu/
+- /docs/connectors/table/kudu/overview.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Apache Kudu
+
+[Apache Kudu](https://kudu.apache.org) is a column-oriented, distributed data storage engine, specifically designed for use cases that require fast analytics on fast (rapidly changing) data. It lowers query latency significantly for Apache Flink and similar engines (Apache Impala, Apache NiFi, Apache Spark, and more).
+
+This connector allows reading and writing to [Kudu](https://kudu.apache.org) by providing:
+
+- a source (```KuduInputFormat```),
+- a sink (```KuduSink```) and output (```KuduOutputFormat```)
+- a table source (`KuduTableSource`),
+- an upsert table sink (`KuduTableSink`),
+- and a catalog (`KuduCatalog`).
+
+It is also possible to use the Kudu connector directly from the DataStream API, however we
+encourage all users to explore the [Table API](#sql-and-table-api) as it provides a lot of useful tooling when working
+with Kudu data.
+
+## Installing Kudu
+
+Follow the instructions from the [Kudu Installation Guide](https://kudu.apache.org/docs/installation.html).
+Optionally, you can use the docker images provided in dockers folder.
+
+## Dependencies
+
+To use this connector, add the following dependency to your project:
+
+{{< connector_artifact flink-connector-kudu kudu >}}
+
+*Version Compatibility*: The current version of the connector is built with Kudu client version **1.17.1**.
+
+Note that the streaming connectors are not part of the binary distribution of Flink. You need to link them into your job jar for cluster execution.
```

Comment:
Lets simplify this a bit. Technically, there are some connectors that are part of the binary distribution (`flink-connector-datagen`, `flink-connector-files`), so lets just say Kudu connector is not part of it.

Also pls. not refer the exact `stable` documentation URL anywhere, but use a dynamic reference like [here](https://github.com/apache/flink-connector-hbase/blob/ca5a4aae3e953841371e9a568ad8472a5ef8c1d2/docs/content/docs/connectors/table/hbase.md?plain=1#L44).

That way the version will be matched during site building.